src/Microsoft.Data.Analysis/PrimitiveDataFrameColumn.BinaryOperations.cs

```diff
@@ -380,6 +380,8 @@ public override PrimitiveDataFrameColumn<bool> ElementwiseEquals(DataFrameColumn
                     return ElementwiseEqualsImplementation(column as PrimitiveDataFrameColumn<byte>);
```

Comment:
I realized I manually edited this file. If it's okay to have DateTime as a primitive dataframe I will update the .tt for this.